app/models/curiosity_scraper.rb

```diff
@@ -0,0 +1,42 @@
+class CuriosityScraper
+  BASE_URL = "http://mars.jpl.nasa.gov/msl/multimedia/raw/"
+
+  def initialize
+    @rover = Rover.find_by(name: "Curiosity")
+  end
+
+  def scrape
+    create_photos
+  end
+
+  # grabs the HTML from the main page of the curiosity rover image gallery
+  def main_page
+    Nokogiri::HTML(open("http://mars.jpl.nasa.gov/msl/multimedia/raw/"))
+  end
+
+  def collect_links
+    # collects link suffixes to pages for martian solar cycle from each camera
+    main_page.css("div.image_list a").map { |link| link['href'] }
+  end
+
+  def create_photos
+    collect_links.each do |url|
+      image_page = Nokogiri::HTML(open(BASE_URL + url))
+      image_array = image_page.css("div.RawImageCaption div.RawImageUTC a")
+        .map { |link| link["href"] }
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.